ibench/benchmarks/bench.py

```diff
@@ -34,28 +35,41 @@ def _run(self, n):
 
         return times
 
-    def measure(self, n):
+    def measure(self, n, args):
         self._log('')
         self._log('  N = %d' % n)
 
         times = self._run(n)
 
         ops = self._ops(n)
-        for elapsed in times:
-            self._log('  elapsed %f gflops %f' % (elapsed,ops/elapsed))
+        if 'gflops' in args:
```

Comment:
I don't understand the logic. If gflops is in args, it doesn't print gflops? Should it be nogflops?

Reply:
Fixed in commit 1e819a4.